app/controllers/jobseekers/job_applications/qualifications_controller.rb

```diff
@@ -49,7 +49,7 @@ def form_attributes
     when "edit"
       qualification
         .slice(:category, :finished_studying, :finished_studying_details, :grade, :institution, :name, :subject, :year, :qualification_results)
-        .reject { |_, v| v.blank? }
+        .reject { |_, v| v.blank? && v != false }
```

Comment:
`false.blank?` returns `true` so the edit pages were missing the `finished_studying` attribute.